firmware/arduino/chessboard/movementFunctions.ino

```diff
@@ -72,7 +72,7 @@ bool moveDirect(uint8_t startCol, uint8_t startRow, uint8_t endCol, uint8_t endR
     Serial.print(", ");
     Serial.print(endRow);
     Serial.print(")");
-    Serial.print("    ");
+    Serial.println("    ");
```

Comment:
You can delete this and make the one above a println and it’ll do the same thing